lib/msal-common/src/response/AuthenticationResult.ts

```diff
@@ -17,6 +17,7 @@ export class AuthenticationResult {
     tokenType: string; // TODO: get rid of this if we can
     idToken: string;
     idTokenClaims: StringDict;
+    refreshToken: string;
```

Comment:
This is temporary and will change soon as we should not send the `refreshToken` to the client. I will merge a new PR defining an `external response` and an internal handler where we need this data to be cached.

Reply:
Removed `refreshToken` now.